webapp/components/generic/SearchableInput/SearchableInput.vue

```diff
@@ -0,0 +1,227 @@
+<template>
+  <div
+    class="searchable-input"
+    aria-label="search"
+    role="search"
+    :class="{
+      'is-active': isActive,
+      'is-open': isOpen,
+    }"
+  >
+    <div class="field">
+      <div class="control">
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T12:07:53Z" title="Wednesday, January 8th 2020, 1:07:53 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,236 @@
+<template>
+  <div
+    class="searchable-input"
+    aria-label="search"
+    role="search"
+    :class="{
+      'is-active': isActive,
+      'is-open': isOpen,
+    }"
+  >
+    <div class="field">
+      <div class="control">
```


That's a lot of wrappers! One should be enough to position the `ds-button` and `ds-select`, no? :wink:

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T12:09:01Z" title="Wednesday, January 8th 2020, 1:09:01 pm +01:00">Jan 8, 2020</time>_

---


I would even prefer this to be an `<input>` rather than a `<div>` but since our beloved styleguide adds a bunch of its own wrapper `divs` it would probably mess things up... :laughing: